### PR TITLE
Added Support for 64bit RGB format

### PR DESCRIPTION
**Pull request by pyn1**
Jira: https://jira01.devtools.intel.com/browse/OAM-52530
Tests: Test Passes
Signed-off-by: Poornima <poornima.y.n@intel.com>

### CODE REVIEW

**Comment by pyn1** [comment]
@kalyankondapally Please review .

File: drv.c [hunk 1]
```diff
@@ -603,6 +603,8 @@ size_t drv_num_planes_from_format(uint32_t format)
 	case DRM_FORMAT_XRGB8888:
 	case DRM_FORMAT_YUYV:
 	case DRM_FORMAT_YVYU:
+	case DRM_FORMAT_XRGB161616:
```
Comment (kalyankondapally) [inline drv.c]: You can add it in i915_private_num_planes_from_format

File: helpers.c [hunk 2]
```diff
@@ -99,6 +99,9 @@ static uint32_t bpp_from_format(uint32_t format, size_t plane)
 	case DRM_FORMAT_XRGB2101010:
 	case DRM_FORMAT_XRGB8888:
 		return 32;
+	case DRM_FORMAT_XRGB161616:
```
Comment (kalyankondapally) [inline helpers.c]: You can add it in i915_private_bpp_from_format

**Comment by pyn1** [comment]
Updated taking care of review comments.
Thanks

**Pull request closed without merging**